docs/docs/faq/faq.md

```diff
@@ -0,0 +1,67 @@
+### Docker-Compose instead of Docker Compose
+
+If you have installed the version of `docker-compose` instead of
+`docker compose`, you might have to do some changes in the commands:
+
+```bash
+    # docker compose up backend-dev --build --attach-dependencies
+    docker-compose up --build --attach-dependencies backend-dev     # starting only backend-dev
+
+    # docker compose up --build
+    docker-compose up --build                                       # starting the all services
+```
```

Comment:
```suggestion
If you are using `docker-compose` instead of `docker compose` (note the " " instead of the "-"), you should update your docker cli to the latest version. `docker compose` is the most recent version and should be used instead of `docker-compose`

For more details and information check out [this SO thread](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose) that explains it all in detail.
```

Reply:
@Nil-Andreu i think the advice here is to just update the docker version to latest. Some people have gotten `docker-compose` to work but just easier (hopefully) for everyone if users can just use `docker compose` instead.

Reply:
Okay completely fine for me thanks!